roles/deploy/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+- name: Initialize
+  deploy_helper: path="{{ project_root }}" state=present
+
+- name: Clone project files
+  git: repo="{{ project_git_repo }}"
+       dest="{{ project_source_path }}"
+       version="{{ project_version }}"
+       accept_hostkey=yes
+
+- name: write unfinished file
+  file: path="{{ project_source_path }}/{{ deploy_helper.unfinished_filename }}" state=touch
+
+- name: Copy files to new build dir
+  command: cp -pr {{ project_source_path }} {{ deploy_helper.new_release_path }}
+
+- name: Move site into root
+  shell: mv {{ deploy_helper.new_release_path }}/{{ project_subtree }}/* {{ deploy_helper.new_release_path }}
+  when: project_subtree is defined
```

Comment:
I don't understand this "Move site into root" task. For me, it just skips because `project_subtree` is not defined.